source/codegen/metadata/nidmm/functions.py

```diff
@@ -0,0 +1,3129 @@
+# -*- coding: utf-8 -*-
+# This file is generated from NI-DMM API metadata version 19.6.0d7
+functions = {
+    'Abort': {
+        'documentation': {
+            'description': '\nAborts a previously initiated measurement and returns the DMM to the\nIdle state.\n'
+        },
+        'parameters': [
+            {
+                'direction': 'in',
+                'documentation': {
+                    'description': '\nIdentifies a particular instrument session. You obtain the **vi**\nparameter from niDMM_init or niDMM_InitWithOptions. The default is\nNone.\n'
+                },
+                'name': 'vi',
+                'type': 'ViSession'
+            }
+        ],
+        'returns': 'ViStatus'
+    },
+    "CalAdjustGain": {
+        'codegen_method': 'public',
```

Comment:
'codegen_method' tags must only be present for APIs that have to be custom implemented.

Reply:
removed codegen_method tags